docs/api/config.md

```diff
@@ -49,6 +49,62 @@ traffic and state size.
 You will also need to specify a directory to write the state data using
 `MESOP_STATE_SESSION_BACKEND_FILE_BASE_DIR`.
 
+#### SQL
+
+> NOTE: Setting up and configuring databases is out of scope of this document.
+
+This option uses [SqlAlchemy](https://www.sqlalchemy.org/) to store Mesop state sessions
+in supported SQL databases, such as SQLite3 and PostgreSQL. You can also connect to
+hosted options, such as GCP CloudSQL.
+
+If you use SQLite3, you cannot use an in-memory database. It has to be a file. This
+option has similar pros/cons as the `file` backend. Mesop uses the default configuration
+for SQLite3, so the performance will not be optimized for Mesop's usage patterns.
+SQLite3 is OK for development purposes.
+
+Using a database like PostgreSQL will allow for better scalability, both vertically and
+horizontally, since the database is decoupled from the Mesop server.
+
+The drawback here is that this requires knowledge of the database you're using. At
+minimum, you will need to create a database and a database user with the right
+privileges. You will also need to create the database table, which you can create
+with this script. You will need to update the CONNECTION_URI and TABLE_NAME to match
+your database and settings. Also the database user for this script will need privileges
+to create tables on the target database.
+
+```python
+from sqlalchemy import (
+  Column,
+  DateTime,
+  LargeBinary,
+  MetaData,
+  String,
+  Table,
+  create_engine,
+)
+
+CONNECTION_URI = "your-database-connection-uri"
+TABLE_NAME = "your-table-name" # or the default "mesop_state_session"
+
+self.db = create_engine(CONNECTION_URI)
```

Comment:
a little confusing why there's self even though this isn't in a class __init__, I would just remove all the self.